package-res/resources/web/pentaho/lang/Base.js

```diff
@@ -35,10 +35,17 @@
  * 11. Class.valueOf removed. No longer needed... ?
  * 12. `Base#__root_proto__` is a new constant, non-enumerable property, set at each "Base" root prototype.
  * 13. `Base#__init__` is a new constant, non-enumerable, property, set at the prototype of "Base" classes
- *      that have an own initialization method (specified with the `constructor` property).
+ *     that have an own initialization method (specified with the `constructor` property).
  * 14. Added new Class.implementStatic method to allow to specify the class interface after extend.
  *     Is parallel to Class.implement.
  * 15. Any existing static methods are inherited (not only standard Base ones).
+ * 16. Dropped support for the overload Base#extend(name, value).
+ *     The same method now supports only the signature Base#extend(instSpec[, keyArgs]).
+ * 17. Added support to augment the set of per-class, inherited property names that are excluded from
+ *     instance-extension operations.
+ *     Specify `extend_exclude` when sub-classing, e.g. `Base.extend({extend_exclude: {a: true, b: true}});`.
+ * 18. Added support to control, per-class, the properties extension order in instance-extension operations.
```

Comment:
What's the use case for this? Did you run into this when implementing something in particular?